### PR TITLE
Add new 4x Incubation Members

### PR DESCRIPTION
**Pull request by szwed**
LinkedIn:

- https://www.linkedin.com/in/wkocjan
- https://www.linkedin.com/in/andrzejszeszo
- https://www.linkedin.com/in/cwienczek
- https://www.linkedin.com/in/pawelmincer

### CODE REVIEW

**Review by mad-it — APPROVED** [review]
Work with them, all great people! 👍🏼 👍🏼

**Comment by kirek007** [comment]
💯 Recommend

**Review by kirek007 — APPROVED** [review]

**Review by moredhel — APPROVED** [review]

File: content/members/andrzej-szeszo/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: "Andrzej Szeszo"
+thumbnail: "images/profile.jpg"
```
Comment (etiennetremel) [inline content/members/andrzej-szeszo/index.md:3]:
```suggestion
thumbnail: "profile.jpg"
```

File: content/members/michael-cwienczek/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Michael Cwienczek"
+thumbnail: "images/profile.jpg"
```
Comment (etiennetremel) [inline content/members/michael-cwienczek/index.md:3]:
```suggestion
thumbnail: "profile.jpg"
```

File: content/members/pawel-mincer/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Pawel Mincer"
+thumbnail: "images/profile.jpg"
```
Comment (etiennetremel) [inline content/members/pawel-mincer/index.md:3]:
```suggestion
thumbnail: "profile.jpg"
```

File: content/members/wojciech-kocjan/index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Wojciech Kocjan"
+thumbnail: "images/profile.jpg"
```
Comment (etiennetremel) [inline content/members/wojciech-kocjan/index.md:3]:
```suggestion
thumbnail: "profile.jpg"
```

**Review by etiennetremel — APPROVED** [review]

**Pull request merged**